examples/markdown/index.html

```diff
@@ -120,6 +120,11 @@
   the content was written in simple Markdown.
 </div>
 
+<div class="step slide title markdown" data-rotate="-85" data-x="4000" data-y="2000" data-extras-markdown-dialect="Maruku">
+This is an example of a slide with a specific Markdown dialect.
```

Comment:
Hmm...

You have done the correct thing, but now I'm thinking maybe we should just call the attribute `data-markdown-dialect`.

Reply:
i like the second one more, just did.